### PR TITLE
PSP-3662: Enforce character limits on FE - Properties

### PR DESCRIPTION
**Pull request by Maheshbabu28**

### CODE REVIEW

File: frontend/src/features/properties/map/research/add/AddResearchFileYupSchema.ts [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
 
 export const AddResearchFileYupSchema = Yup.object().shape({
-  name: Yup.string().required('Research File name is required'),
-  properties: Yup.array(),
+  name: Yup.string()
+    .required('Research File name is required')
+    .max(500, 'Research File name must be less than 500 characters'),
+  properties: Yup.array().of(
+    Yup.object().shape({
+      name: Yup.string().max(250, 'Property name must be less than 500 characters'),
```
Comment (devinleighsmith) [inline frontend/src/features/properties/map/research/add/AddResearchFileYupSchema.ts:9]: 500 -> 250

File: frontend/src/features/properties/map/research/update/UpdateResearchFileYupSchema.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import * as Yup from 'yup';
+
+export const UpdateResearchFileYupSchema = Yup.object().shape({
+  name: Yup.string().max(250, 'Research File name must be less than 500 characters'),
```
Comment (devinleighsmith) [inline frontend/src/features/properties/map/research/update/UpdateResearchFileYupSchema.ts:4]: 500 -> 250

**Comment by devinleighsmith** [comment]
@ please correct comments before merging

**Review by devinleighsmith — CHANGES_REQUESTED** [review]
please address comments

File: frontend/src/features/properties/map/research/add/AddResearchFileYupSchema.ts (hunk 1 above)
Comment (Maheshbabu28) [reply frontend/src/features/properties/map/research/add/AddResearchFileYupSchema.ts]: Updated it with 250

File: frontend/src/features/properties/map/research/update/UpdateResearchFileYupSchema.ts (hunk 2 above)
Comment (Maheshbabu28) [reply frontend/src/features/properties/map/research/update/UpdateResearchFileYupSchema.ts]: Updated it with 250

**Pull request merged**